ShowContact/index.ts

```diff
@@ -27,5 +28,4 @@ const httpTrigger: AzureFunction = async function (
     });
   }
 };
-
```

Comment:
we use a popular library lodash which has helper functions ie allow us to pick properties we want from an object. https://lodash.com/docs/4.17.15#pick

You should try use that